src/api/docker-compose-api-production.yml

```diff
@@ -156,19 +156,52 @@ services:
       traefik:
         condition: service_started
     ports:
-      - '4444'
+      - "4444"
     labels:
       # Traefik routing
-      - 'traefik.http.routers.image.rule=Host(`image.docker.localhost`)'
+      - "traefik.http.routers.image.rule=Host(`image.docker.localhost`)"
       # Enable gzip compression
-      - 'traefik.http.routers.image.middlewares=test-compress'
-      - 'traefik.http.middlewares.test-compress.compress=true'
+      - "traefik.http.routers.image.middlewares=test-compress"
+      - "traefik.http.middlewares.test-compress.compress=true"
       # ELK Logging
-      - 'co.elastic.logs/json.keys_under_root: true'
-      - 'co.elastic.logs/json.overwrite_keys: true'
-      - 'co.elastic.logs/json.add_error_key: true'
-      - 'co.elastic.logs/json.expand_keys: true'
-      - 'co.elastic.logs/json.message_key: message'
+      - "co.elastic.logs/json.keys_under_root: true"
+      - "co.elastic.logs/json.overwrite_keys: true"
+      - "co.elastic.logs/json.add_error_key: true"
+      - "co.elastic.logs/json.expand_keys: true"
+      - "co.elastic.logs/json.message_key: message"
+
+  search:
+    container_name: "search"
+    restart: unless-stopped
+    build:
+      context: ./search
+      dockerfile: Dockerfile
+    environment:
+      - NODE_ENV=production
+      - IMAGE_PORT=4445
+      - SERVICE_NAME=search
```

Comment:
Change this to `ELASTIC_APM_SERVICE_NAME` to match how I'm doing it in 1.5.x of satellite.